pkg/core/interop_neo_test.go

```diff
@@ -231,6 +233,67 @@ func TestWitnessGetVerificationScript(t *testing.T) {
 	require.Equal(t, witness.VerificationScript, value)
 }
 
+func TestECDSAVerify(t *testing.T) {
```

Comment:
BTW, are you sure this test would count for coverage in `interop/crypto` after the package split?

Reply:
Yes, it counts.

I'd like to move this test to an `interpop/crypto`, but this requires a substantial refactoring:
1. Move `newTestChain` to an `internal` package.
2. All packages requiring `newTestChain` must have test in `*_test` packages to avoid import cycle.

Reply:
I was talking about local `go test`. It seems that `codecov` indeed does not count for coverage.

Reply:
I think there is a better option. If we're to solve this:
```
go build github.com/nspcc-dev/neo-go/integration: no non-test Go files in /home/rik/dev/neo-go/integration
```
by moving or removing this test, then we could add `-coverpkg=all` flag and that should help with cross-package coverage.